.gitignore

```diff
@@ -146,3 +146,8 @@ npm-debug.log*
 static/dist
 derby.log
 metastore_db
+
+airflow-*.err
+airflow-*.out
+airflow-*.log
+airflow-*.pid
```

Comment:
My understanding is these files are generated only when we run Airflow. But do we need to add them here?
If yes, maybe also necessary to add a short description of category like what have been added for other entries above.

Reply:
Yeah. They are generated when we run various Airflow tests using IDE (in some scenarios the files are generated here - rather than in $HOME/airflow. I did not check exactly when it happens but I figured it makes sense to add it to .gitignore so that we don't commit it accidentally.

Reply:
Added comment explaining it.